src/common/meta/src/cache/container.rs

```diff
@@ -101,9 +101,15 @@ where
 {
     /// Returns a _clone_ of the value corresponding to the key.
     pub async fn get(&self, key: K) -> Result<Option<V>> {
```

Comment:
Can we also provide a `batch_get`? But not in this PR. We have already implemented this in Cached KvBackend: https://github.com/GreptimeTeam/greptimedb/blob/89da42dbc12687157e00dbd4efae75ba154381c1/src/catalog/src/kvbackend/client.rs#L170

Reply:
It's a little tricky, we need to employ a lock to ensure consistency.